packages/api/server/http.mts

```diff
@@ -104,17 +87,24 @@ router.delete('/srcbooks/:id', cors(), async (req, res) => {
 // Import a srcbook from a .src.md file or srcmd text.
 router.options('/import', cors());
 router.post('/import', cors(), async (req, res) => {
-  const { path, text } = req.body;
+  const { path, text, url } = req.body;
 
   if (typeof path === 'string' && !isSrcmdPath(path)) {
     return res.json({ error: true, result: 'Importing only works with .src.md files' });
   }
+  if (typeof url === 'string' && !isSrcmdPath(url)) {
+    return res.json({ error: true, result: 'Importing only works with .src.md files' });
+  }
 
   try {
     if (typeof path === 'string') {
       posthog.capture({ event: 'user imported srcbook from file' });
       const srcbookDir = await importSrcbookFromSrcmdFile(path);
       return res.json({ error: false, result: { dir: srcbookDir } });
+    } else if (typeof url === 'string') {
+      posthog.capture({ event: 'user imported srcbook from url' });
+      const srcbookDir = await importSrcbookFromSrcmdUrl(url);
```

Comment:
yessssss. Do you mind making a follow up note once this PR goes through to update [the CLI code](https://github.com/srcbookdev/srcbook/blob/main/srcbook/src/cli.mts#L81-L131) to use this?

Reply:
Unless you object, I'll just include that change in this pull request - it doesn't seem that complex, and I'd rather just get it done.

Reply:
Sounds good

Reply:
FWIW, I think this change should be here: https://github.com/srcbookdev/srcbook/pull/307/commits/4892f66048957558c644e37d4851a963e7e982f1